### PR TITLE
Closes #1 - Create initial index.html markup

### LINKED ISSUE #1: Create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

**Comment by simonpals** [comment]
Please add changes at the end of file

**Review by KrilOstap — CHANGES_REQUESTED** [review]
Please add a new line the end of file

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (ttymchuk) [inline index.html]: Please add empty line.

Comment (OlehSihunov) [inline index.html]: add empty line

Comment (dashakrashchenko) [inline index.html]: Please add empty line

Comment (andriy-mudrak) [inline index.html]: please add empty line

**Review by OlehSihunov — CHANGES_REQUESTED** [review]
see comments

File: index.html (hunk 1 above)
Comment (yurapav99) [inline index.html]: please, add empty line

Comment (vdiakiv) [inline index.html]: Please add empty line

**Review by dashakrashchenko — CHANGES_REQUESTED** [review]
watch comments

File: index.html (hunk 1 above)
Comment (taras-tymchuk) [inline index.html]: Please add empty line

**Review by yurapav99 — CHANGES_REQUESTED** [review]
please, check it

**Review by taras-tymchuk — CHANGES_REQUESTED** [review]

**Review by vladmamai — CHANGES_REQUESTED** [review]
Please add empty row at the end of file.

**Review by YarishPetro — COMMENTED** [review]
Add new line

**Review by vdiakiv — CHANGES_REQUESTED** [review]
Check comments

**Review by andriy-mudrak — COMMENTED** [review]
Please add empty line

**Review by vladmamai — APPROVED** [review]

**Review by YarishPetro — CHANGES_REQUESTED** [review]
Add new line in the end

**Review by andriy-mudrak — CHANGES_REQUESTED** [review]
Please add empty line.

File: index.html (hunk 1 above)
Comment (simonpals) [inline index.html]: Fix it

**Review by simonpals — CHANGES_REQUESTED** [review]

**Review by taras-tymchuk — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (taras-tymchuk) [inline index.html]: enpty line

**Review by taras-tymchuk — CHANGES_REQUESTED** [review]

**Review by vladmamai — CHANGES_REQUESTED** [review]
add empty line

**Review by KrilOstap — APPROVED** [review]

**Review by YarishPetro — APPROVED** [review]

**Review by simonpals — APPROVED** [review]
Good job

**Review by taras-tymchuk — APPROVED** [review]

**Review by andriy-mudrak — APPROVED** [review]
good job

**Review by vdiakiv — APPROVED** [review]

**Review by yurapav99 — APPROVED** [review]
well done!

**Review by vladmamai — APPROVED** [review]

**Review by dashakrashchenko — APPROVED** [review]

**Review by OlehSihunov — APPROVED** [review]

**Pull request merged**